firstvoices/firstvoices/settings.py

```diff
@@ -120,6 +120,55 @@
 # LOGGERS
 ELASTICSEARCH_LOGGER = "elasticsearch"
 
+# Based on the default settings
+# ref: https://docs.djangoproject.com/en/5.1/ref/logging/#default-logging-configuration
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "filters": {
+        "debug_info_log_level_filter": {
+            "()": "firstvoices.logging_utils.DebugInfoLogLevelFilter"
+        },
+    },
+    "formatters": {
+        "standard": {
+            "format": "{levelname}-{asctime}:{pathname} -- {message}",
+            "style": "{",
+        },
+        "detailed": {
+            "format": "{levelname}-{asctime}:{pathname} {process:d} {thread:d} -- {message}",
+            "style": "{",
+        },
+    },
+    "handlers": {
+        "console": {
+            "level": "DEBUG",  # DEBUG + INFO
+            "filters": ["debug_info_log_level_filter"],
+            "class": "logging.StreamHandler",
+            "formatter": "standard",
+        },
+        "file": {
+            "level": "WARNING",  # WARNING + ERROR
+            "class": "logging.FileHandler",
+            "filename": "./app.log",
```

Comment:
Let's not keep the log files in root. Cleanup schedules can get messy. 

/var/log/django/app.log would be fine I guess.